rest-api/src/main/java/gov/cms/qpp/conversion/api/services/ValidationServiceImpl.java

```diff
@@ -39,10 +37,13 @@
 	private static final Logger API_LOG = LoggerFactory.getLogger(Constants.API_LOG);
 	static final String CONTENT_TYPE = "application/json";
 
-	@Autowired
 	private Environment environment;
+	private RestTemplate restTemplate;
 
-	private RestTemplate restTemplate = new RestTemplate();
+	public ValidationServiceImpl(final Environment environment) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public constructor by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)

Reply:
Also, I agree.  Let's add JavaDocs here.